### ISSUE TITLE
Cheryl has Trash TCPA Rebuttals

### ISSUE THREAD

**Issue by MTS11648** [LCN-Dev/GoogleBot#179]
#### Brief description of the issue:
Cheryl's only rebuttal to the TCPA is of her screaming into the microphone, implying the customer is an idiot if they don't take advantage of these great deals and steals.
#### What happened?:
Customer says no to the TCPA. Cheryl goes from 0 to 100 and yells at them.
#### What _should_ have happened?:
Cheryl kindly tells the customer why they should consider getting the quote and gets progressively angrier at the customer's negligence to consent.
#### Steps to reproduce:
1) Make it to the TCPA.
2) Say, "No."
3) Watch Cheryl burst into flames, telling you why you would be foolish to not take advantage of the amazing quote opportunity.
#### Additional info:
TLDR: I want to have Cheryl only get angry after the customer fails to consent several times in a row. Her first rebuttal will make her sound understanding, but if the customer keeps neglecting to consent, she will begin to lay down the "big guns".

**Comment by NicksAccount** [comment]
Working on going thru Cheryl's calls, specifically LOWs to find some. 

Here's 1 real one and 1 comedic (and actual) tcpa rebuttal by Cheryl. 
[tcpa rebuttz.zip](https://github.com/LCN-Dev/GoogleBot/files/916851/tcpa.rebuttz.zip)


The search continues.
